templates/taxi/driver_form.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+{% block content %}
+  <h1>{{ object|yesno:"Update,Create" }} driver</h1>
```

Comment:
The `yesno` filter is used to toggle between 'Update' and 'Create' based on the presence of an `object`. Ensure that the context passed to this template includes an `object` when updating a driver, so the correct heading is displayed.